pkg/config/justification_config.go

```diff
@@ -45,6 +46,9 @@ type JustificationConfig struct {
 	// Issuer will be used to set the issuer field when signing JWTs
 	Issuer string `env:"JVS_API_ISSUER,overwrite,default=jvs.abcxyz.dev"`
 
+	// PluginAbsDir is the absolute path of the directory to load plugins.
+	PluginAbsDir string `env:"JVS_PLUGIN_ABS_DIR,overwrite,default=/var/jvs/plugins"`
```

Comment:
Is there any particular reason to require abs dir?

Reply:
See related conversation from https://github.com/abcxyz/jvs/pull/294#discussion_r1247892221, my guess is it is easier to manage the dir structure of the image. Looping in @sethvargo for more information

Reply:
Done